httputil/logger.go

```diff
@@ -55,22 +55,22 @@ func (f debugflag) debugBody() bool {
 type LoggedTransport struct {
 	Transport http.RoundTripper
 	Key       string
-	body      bool
+	Body      bool
```

Comment:
It wasn't clear what this was from the `Body: true` usage. Maybe `LogBody` or `MayLogBody`?